test/e2e/e2e.go

```diff
@@ -399,25 +399,25 @@ func lookupClusterImageSources() (string, string, error) {
 		nodeName = lines[0]
 	}
 
-	nodeImg, err := host2image(nodeName)
+	dataPlaneImg, err := host2image(nodeName)
```

Comment:
`dataPlaneNodeImg` instead for consistency?

Reply:
good point, will fix ^